test/kwtool_web/controllers/upload_controller_test.exs

```diff
@@ -15,4 +15,34 @@ defmodule KwtoolWeb.UploadControllerTest do
       assert html_response(conn, 200) =~ "Upload"
     end
   end
+
+  describe "post create/2" do
```

Comment:
This test missing an invalid file type case.

Reply:
yes, added